proparse/src/main/java/org/prorefactor/treeparser/TreeParserRootSymbolScope.java

```diff
@@ -50,10 +56,10 @@ public void addTableDefinitionIfNew(ITable table) {
   }
 
   /** Assign a super (inherited) class scope to this class scope. */
-  public void assignSuper(SymbolScopeRoot superScope) {
+  /* public void assignSuper(SymbolScopeRoot superScope) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)